tools/StaticAnalysis/Exceptions/Az.Network/SignatureIssues.csv

```
@@ -338,4 +338,5 @@
 "Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.NetworkWatcher.NewNetworkWatcherConnectionMonitorProtocolConfigurationObject","New-AzNetworkWatcherConnectionMonitorProtocolConfigurationObject","1","8100","New-AzNetworkWatcherConnectionMonitorProtocolConfigurationObject Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
 "Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.NetworkWatcher.NewNetworkWatcherConnectionMonitorProtocolConfigurationObject","New-AzNetworkWatcherConnectionMonitorProtocolConfigurationObject","1","8510","Cmdlet 'New-AzNetworkWatcherConnectionMonitorProtocolConfigurationObject' has multiple parameter sets, but no defined default parameter set.","Define a default parameter set in the cmdlet attribute."
-"Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.RemoveAzureSecurityPartnerProviderCommand","Remove-AzSecurityPartnerProvider","1","8510","Cmdlet 'Remove-AzSecurityPartnerProvider' has multiple parameter sets, but no defined default parameter set.","Define a default parameter set in the cmdlet attribute."
\ No newline at end of file
+"Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.RemoveAzureSecurityPartnerProviderCommand","Remove-AzSecurityPartnerProvider","1","8510","Cmdlet 'Remove-AzSecurityPartnerProvider' has multiple parameter sets, but no defined default parameter set.","Define a default parameter set in the cmdlet attribute."
```

Please do not suppress the error. You can set a default parameter for "GetAzureSecurityPartnerProviderCommand".